src/components/errorfunc.js

```diff
@@ -0,0 +1,48 @@
+//function to check whether the url is correct
```

Comment:
You'll probably need to scrap all of this code and it's usage in other components as it isn't what is required currently :(